src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -287,9 +287,9 @@ class AutomaticSpeechRecognitionPipeline(ChunkPipeline):
             installed. If no framework is specified, will default to the one currently installed. If no framework is
             specified and both frameworks are installed, will default to the framework of the `model`, or to PyTorch if
             no model is provided.
-        device (`int`, *optional*, defaults to -1):
-            Device ordinal for CPU/GPU supports. Setting this to -1 will leverage CPU, a positive will run the model on
-            the associated CUDA device id.
+        device (Union[`int`, `torch.device`], *optional*, defaults to `None`):
```

Comment:
```suggestion
        device (Union[`int`, `torch.device`], *optional*):
```
Don't write the default if it's None ;-)

Reply:
Ah yes! :D